src/_index.js

```diff
@@ -121,7 +121,7 @@ export class EasyTabAccordion{
         this.options.animation = animationValue !== null ? animationValue : this.options.animation;
 
         // get options init by data attribute (JSON format)
-        getOptions(this);
+        this.options = getOptions(this, this.options);
```

Comment:
> `getOptions(this, this.options)`

We can get `this.options` from `this`, right? Could you remove `this.options` please?

Reply:
I have changed it, more detail at https://github.com/viivue/easy-tab-accordion/pull/27/commits/8cc245e6c57c1db6050dfad87365bd80ce49a2f2

But why I put the second parameter (defaultOptions) in here is because we can store options by using many names (options, config,...). So for reuse purposes, I have put a second parameter to solve it.